lib/signs_ui_web/controllers/signs_controller.ex

```diff
@@ -15,71 +21,14 @@ defmodule SignsUiWeb.SignsController do
-      {"world_trade_center_outbound", "World Trade Center Outbound"},
-      {"world_trade_center_mezzanine", "World Trade Center Mezzanine"},
-      {"courthouse_station_mezzanine", "Courthouse Station Mezzanine"},
-      {"south_station_mezzanine", "South Station Mezzanine"},
-      {"wonderland_westbound", "Wonderland Westbound"},
-      {"wonderland_mezzanine", "Wonderland Mezzanine"},
-      {"revere_beach_eastbound", "Revere Beach Eastbound"},
-      {"revere_beach_mezzanine_westbound", "Revere Beach Mezzanine Westbound"},
-      {"revere_beach_mezzanine_eastbound", "Revere Beach Mezzanine Eastbound"},
-      {"revere_beach_westbound", "Revere Beach Westbound"},
-      {"beachmont_westbound", "Beachmont Westbound"},
-      {"beachmont_eastbound", "Beachmont Eastbound"},
-      {"suffolk_downs_eastbound", "Suffolk Downs Eastbound"},
-      {"suffolk_downs_westbound", "Suffolk Downs Westbound"},
-      {"orient_heights_eastbound", "Orient Heights Eastbound"},
-      {"orient_heights_mezzanine_westbound", "Orient Heights Mezzanine Westbound"},
-      {"orient_heights_mezzanine_eastbound", "Orient Heights Mezzanine Eastbound"},
-      {"orient_heights_westbound", "Orient Heights Westbound"},
-      {"wood_island_eastbound", "Wood Island Eastbound"},
-      {"wood_island_mezzanine_westbound", "Wood Island Mezzanine Westbound"},
-      {"wood_island_mezzanine_eastbound", "Wood Island Mezzanine Eastbound"},
-      {"wood_island_westbound", "Wood Island Westbound"},
-      {"airport_eastbound", "Airport Eastbound"},
-      {"airport_westbound", "Airport Westbound"},
-      {"maverick_eastbound", "Maverick Eastbound"},
-      {"maverick_westbound", "Maverick Westbound"},
-      {"aquarium_eastbound", "Aquarium Eastbound"},
-      {"aquarium_mezzanine_westbound", "Aquarium Mezzanine Westbound"},
-      {"aquarium_mezzanine_eastbound", "Aquarium Mezzanine Eastbound"},
-      {"aquarium_westbound", "Aquarium Westbound"},
-      {"state_blue_eastbound", "State Eastbound"},
-      {"state_blue_mezzanine_westbound", "State Mezzanine Westbound"},
-      {"state_blue_mezzanine_eastbound", "State Mezzanine Eastbound"},
-      {"state_blue_westbound", "State Westbound"},
-      {"government_center_eastbound", "Government Center Eastbound"},
-      {"government_center_mezzanine_westbound", "Government Center Mezzanine Westbound"},
-      {"government_center_mezzanine_eastbound", "Government Center Mezzanine Eastbound"},
-      {"government_center_westbound", "Government Center Westbound"},
-      {"bowdoin_eastbound", "Bowdoin Eastbound"},
-      {"bowdoin_mezzanine", "Bowdoin Mezzanine"}
-    ]
+  def sign_names(%{"route" => "mattapan"}), do: Signs.Names.mattapan()
+  def sign_names(%{"route" => "blue"}), do: Signs.Names.blue_line()
+  def sign_names(%{"route" => "silver_line"}), do: Signs.Names.silver_line()
+  def sign_names(%{"route" => "all"}) do
+    Signs.Names.mattapan() ++ Signs.Names.silver_line ++ Signs.Names.blue_line()
```

Comment:
For ordering, how about subway before bus, and then subway ordered the way it is on mbta.com? I believe the proper order is specified via the API, but I don't think we need to reach out for that here... it's unlikely to change at this point.